src/app/server/CommissioningWindowManager.cpp

```diff
@@ -405,7 +412,10 @@ CHIP_ERROR CommissioningWindowManager::StopAdvertisement(bool aShuttingDown)
 #if CONFIG_NETWORK_LAYER_BLE
     if (mIsBLE)
     {
-        ReturnErrorOnFailure(chip::DeviceLayer::ConnectivityMgr().SetBLEAdvertisingEnabled(false));
+        // Ignore errors from SetBLEAdvertisingEnabled (which could be due to
+        // BLE advertising not being supported at all).  Our commissioning
+        // window is now closed and we need to notify our delegate of that.
+        chip::DeviceLayer::ConnectivityMgr().SetBLEAdvertisingEnabled(false);
```

Comment:
explicit `(void)` to mark in the code we ignore return value?

Reply:
Can do.